### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Yehorf21**
- [DEMO LINK](https://Yehorf21.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1 +1,84 @@
...
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    box-sizing: border-box;
```
Comment (Viktor-Kost) [inline src/styles/main.scss:20]: move to `*` selector

File: src/styles/main.scss [hunk 2]
```diff
@@ -1 +1,84 @@
...
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 3600s steps(60) infinite running;
+    transform-origin: bottom center;
+    z-index: -2;
+  }
```
Comment (Viktor-Kost) [inline src/styles/main.scss:50]: both hands have a lot of similar styles. Move repeated styles either to a mixin or to a common class selector

File: src/styles/main.scss [hunk 3]
```diff
@@ -1 +1,84 @@
...
+
+  75% {
+    transform: translate(-50%) rotate(270deg);
+  }
+
+  100% {
+    transform: translate(-50%) rotate(360deg);
+  }
+}
```
Comment (Viktor-Kost) [inline src/styles/main.scss:81]: 1. move translate(-50%) outside the animation (to your class selectors)
2. no needs to such a big amount of keyframes
```suggestion
@keyframes rotate {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}
```

File: src/styles/main.scss [hunk 4]
```diff
@@ -1 +1,84 @@
...
+
+
+
```
Comment (Viktor-Kost) [inline src/styles/main.scss:84]:
```suggestion
```

File: src/styles/main.scss [hunk 5]
```diff
@@ -1 +1,84 @@
...
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 60s linear infinite running;
+    transform-origin: bottom center;
+    z-index: -1;
```
Comment (Viktor-Kost) [inline src/styles/main.scss:37]: it's better to use `positive` values (1+)

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Also please check your deployment since DEMO link leads to a blank page. If you need any help feel free to ask at fe_chat

File: src/styles/main.scss (hunk 5 above)
Comment (Yehorf21) [reply src/styles/main.scss:37]: When I use positive values my hands are shown: 

<img width="447" alt="Screenshot 2024-01-30 at 10 23 33 PM" src="https://github.com/mate-academy/layout_stop-watch/assets/148825530/7070ce90-fba7-49ba-a2fa-48034710e9f2"

File: src/styles/main.scss (hunk 3 above)
Comment (Yehorf21) [reply src/styles/main.scss]: I had to leave 25% because tests wouldn't pass otherwise

**Comment by Yehorf21** [comment]
- [DEMO LINK](https://Yehorf21.github.io/layout_stop-watch/)

**Comment by Yehorf21** [comment]
There's a mistake, I'll upload an update a bit later

**Comment by Yehorf21** [comment]
I don't understand why my hands move faster than expected

Screenshot 2024-01-31 at 9 14 53 AM

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, your demo link is empty now, you need to fix it
image

**Pull request closed without merging**